src/cli.ts

```diff
@@ -38,15 +39,19 @@ if (args['--help']) {
     Options
       --version, -v       Version number
       --help, -h          Displays this message
+
+      --gitignore         Respect ignore patterns in '.gitignore' files
```

Comment:
I actually preferred the initial idea with `--no-gitignore` as the default is to use `.gitignore`. I think that the PR changes the default and I assume it's not the intention.

Reply:
My intention was the following:

1. you changed the default behaviour of globby, by assuming gitignore should be true by default, globby however has the default of false
2. prefixing the option with --no-... got me thinking, as it may be a hard to understand double negation and thus cause confusion:

--no-gitignore: does it now mean that the files in gitignore get ignored or included, so is it that "gitignore should NOT be checked" or "NO files from gitignore should be checked"?

Thus I decided to stick to the original parameter, how also globby called it "gitignore" and adaped the default to be as the underlying (much broader used and thus in my opinion, more accepted) library.

Please let me know about your preference.

Reply:
The reason I ignored all files within the gitignore by default is because in most cases those are build related files that you don't really want to work with, in most cases this is why they are not versioned control. The tool should work for 99% of the use-cases without configuration. That doesn't means that we won't allow configuring it.

Regarding the double negation, I agree that it's complex.

I'm ok with having `gitignore=true` as the option that will do the following: "the glob pattern will include files from the .gitignore files". The description should be changed accordingly, or something like that.